documentation/book/snip-images.adoc

```diff
@@ -32,4 +30,11 @@ a|
 * User Operator
 * Kafka Initializer
 
+|Kafka Bridge
+a|
+* {DockerOrg}/kafka-bridge:{DockerTag}
```

Comment:
Since we can release the bridge independently of the operators we logically should have a different attribute here, such as `{BridgeDockerTag}`.

Reply:
:BridgeDockerTag: {ProductVersion}